### PR TITLE
Rke2 september r2

### PR DESCRIPTION
**Pull request by briandowns**

### CODE REVIEW

File: channels-rke2.yaml [hunk 1]
```diff
@@ -630,6 +630,18 @@ releases:
         version: v3.23.3-build2022081001
     featureVersions: &featureVersions-v1
       <<: *featureVersions-v1-22-11-rke2r1
+  - version: v1.22.15-rc1+rke2r2
+    minChannelServerVersion: v2.6.3-alpha1
+    maxChannelServerVersion: v2.7.99
+    serverArgs: *serverArgs-v1-22-11-rke2r1
+    agentArgs: *agentArgs-v1-22-11-rke2r1
+    charts: &charts-v1-22-15-rke2r1
+      <<: *charts-v1
```
Comment (brandond) [inline channels-rke2.yaml]: Incomplete previous version anchor to include from?

Comment (briandowns) [reply channels-rke2.yaml]: Ah, lovely copy/pasta fun.

**Review by rancher-max — DISMISSED** [review]

**Review by brandond — DISMISSED** [review]

File: channels-rke2.yaml [hunk 2]
```diff
@@ -617,7 +617,7 @@ releases:
       <<: *featureVersions-v1-22-11-rke2r1
   - version: v1.22.15+rke2r1
     minChannelServerVersion: v2.6.3-alpha1
-    maxChannelServerVersion: v2.7.99
+    maxChannelServerVersion: v2.6.99
```
Comment (brandond) [inline channels-rke2.yaml:620]: Is this intentional for the dev-v2.7 branch?

Comment (briandowns) [reply channels-rke2.yaml:620]: Yes. Was informed today by @rancher-max that 1.22 will be hidden in 2.7.

**Review by brandond — APPROVED** [review]

**Pull request merged**